types/sequelize/index.d.ts

```diff
@@ -5105,6 +5105,27 @@ declare namespace sequelize {
     interface Options {
 
         /**
+         * The username which is used to authenticate against the database.
```

Comment:
There are three form of the contructor
```
constructor(db,user?,pass?,options?)
constructor(URI, options?)
constructor(options)
```

And it should always contain `dialect` and `host` in the URI, so if follow your suggestion, there should be 3 types of Options. I think it may be a bit complicated.